projects/cashmere-examples/src/lib/pagination-simple/pagination-simple-example.html

```diff
@@ -0,0 +1,5 @@
+<hc-form-field> <hc-label>Total number of items</hc-label> <input hcInput type="number" [(ngModel)]="totalItems" /> </hc-form-field>
```

Comment:
Even with the table examples, I think there is still value in having a non-table example for the pagination component.  I have found myself actually using it without tables more than with tables, and it is nice to see how to set that up.